client/ayon_core/cli.py

```diff
@@ -102,19 +102,18 @@ def extractenvironments(output_json_path, project, asset, task, app, envgroup):
 
 
 @main_cli.command()
-@click.argument("paths", nargs=-1)
-@click.option("-t", "--targets", help="Targets module", default=None,
+@click.argument("path", nargs=1)
```

Comment:
I believe `nargs=1` still makes the `path` variable a `list` but just containing the one entry - which means in effect we're now passing the wrong argument type to `Commands.publish()` because the docstring there is now stating it should be `str` type.